Makefile

```diff
@@ -36,18 +36,14 @@ generate-env-file: ## Generate the environment file for running the tests inside
 
 .PHONY: prepare-docker-runner-image
 prepare-docker-runner-image: ## Prepare the Docker builder image
-	make -C docker build
+	docker pull `grep "FROM " Dockerfile | cut -d ' ' -f 2` || true
```

Comment:
if we're in cleanup we really dont need this really gross docker pull. i'm pretty sure it doesnt do anything that just docker build on its own wont